manifests/params.pp

```diff
@@ -0,0 +1,13 @@
+# == Class: puppetcrl_sync::params
+#
+# A set of default parameters
+#
+class puppetcrl_sync::params {
+
+  $crl_location = '/var/lib/puppet/ssl/crl.pem'
+  $crl_target   = '/var/lib/puppet/ssl/crl.pem'
```

Comment:
No, because the cat_file function is only evaluated on the puppet master. Although you don't want to include this class on the puppet master, which I will add to the README.